presto-main/src/main/java/com/facebook/presto/cost/ScalarStatsCalculator.java

```diff
@@ -99,5 +105,56 @@ protected SymbolStatsEstimate visitLiteral(Literal node, Void context)
             }
             return estimate.build();
         }
+
+        @Override
+        protected SymbolStatsEstimate visitCast(Cast node, Void context)
+        {
+            SymbolStatsEstimate sourceStats = process(node.getExpression());
+            TypeSignature targetType = TypeSignature.parseTypeSignature(node.getType());
+
+            // todo - make this general postprocessing rule.
```

Comment:
This is about using "normalizers" which isn't ready yet. Making it ready and rebasing before here would take a lot of (otherwise not very productive) effort.